tools/requirements.txt

```diff
@@ -1,4 +1,5 @@
 click
+deepdiff
```

Comment:
I think you're not really using this.

Reply:
I do not use it, but it was needed to run pytest in the root folder